x-pack/plugins/cases/public/common/translations.ts

```diff
@@ -108,6 +122,10 @@ export const TAGS = i18n.translate('xpack.cases.caseView.tags', {
   defaultMessage: 'Tags',
 });
 
+export const SOLUTION = i18n.translate('xpack.cases.caseView.solution', {
+  defaultMessage: 'Solution',
```

Comment:
I believe in the mockups the label is `Case Type`, did it change?

Reply:
Yeah it might have been while you were out but Paul asked for it to be `Solution`.

Reply:
Yeah, a conversation with Paul and the team concluded that Solution was a better term. That said theres a conversation happening in the background with design as to whether there is something better we could call it. 😆